src/path.js

```diff
@@ -623,6 +633,13 @@ Path.prototype.toPathData = function(options) {
  * @return {string}
  */
 Path.prototype.toSVG = function(options, pathData) {
+    if (this._layers && this._layers.length) {
```

Comment:
Maybe some inspiration: https://yne.fr/picon/list?flags
Code: https://github.com/yne/picon/blob/master/edit.html

Reply:
Yeah, maybe we should just implement it in a way we see fit, and then adapt the format via the bin script, once the guys over at the unicode test suite decided on a format they'd like to have.